### ISSUE TITLE
Calendar with showTime not updating model when only time has been changed

### ISSUE THREAD

**Issue by bombadillo** [primefaces/primeng#2941]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Instead of a Plunkr Case, the issue can be viewed on the PrimeNG website: https://www.primefaces.org/primeng/#/calendar

**Current behavior**
Clicking the change time buttons in a calendar component with a `showTime` attribute does not change the models (bound ngModel) value until a date is selected. However, it does change the value within the calendar input.

**Expected behavior**
Clicking the change time buttons should update the model to the new value.

**Minimal reproduction of the problem with instructions**
* Go to https://www.primefaces.org/primeng/#/calendar
* Click on "Time" input
* Click buttons to modify time
* Input box is updated
* Model is not updated (bug)

**Comment by 4kochi** [comment]
Same for us. We have a reactive form. For debugging there is an output in the form of `form.getRawValue()`. I can see that the model value is updated in the view, but the reactive form does value get's not update. Therefore the validation is not triggered.

**Comment by AyaSypron** [comment]
any news for this bug?

**Comment by cagataycivici** [comment]
Fixed now for 4.3.0
